pkg/node/node.go

```diff
@@ -969,6 +980,9 @@ func (b *Bee) Shutdown(ctx context.Context) error {
 
 	tryClose(b.apiCloser, "api")
 
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
this was 15s and now is 5s. it needs to be evaluated whether this is the right value, and what are the side effects of a non-clean api shutdown (since this propagates to other components that are later shut-down.